examples/output/cat.html

```diff
@@ -396,9 +396,9 @@ <h3>n_cubes </h3> <p> 15</p>
     
         <h3>perc_overlap </h3> <p> 0.2</p>
     
-        <h3>clusterer </h3> <p> DBSCAN(algorithm='auto', eps=0.1, leaf_size=30, metric='euclidean',<br>    metric_params=None, min_samples=5, n_jobs=None, p=None)</p>
+        <h3>clusterer </h3> <p> DBSCAN(eps=0.1)</p>
```

Comment:
I'm not sure why these got overwritten. I can revert all changes to `examples`. Must have happened when I was building the docs locally.

Reply:
Actually, this would have something to do with the template code that builds the examples. Hmm.

Reply:
I think that sklearn changed the way `__repr__` works and only shows the values that aren't the default.  Not a worry.